.github/workflows/win-exe.yml

```diff
@@ -194,19 +189,14 @@ jobs:
       semVerStr: ${{ steps.versions.outputs.semVerStr }}
       revNum: ${{ steps.versions.outputs.revNum }}
 
-  publish-winget:
-    name: Publish on winget repo
-    runs-on: windows-latest
+  call-winget-flow:
     needs: [build-msi]
-    if: github.event.action == 'published' || inputs.winget-release
-    steps:
-      - name: Submit package to Windows Package Manager Community Repository
-        run: |
-          iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
-          $github = Get-Content '${{ github.event_path }}' | ConvertFrom-Json
-          $installerUrl = $github.release.assets | Where-Object -Property name -match '^Cryptomator-.*\.msi' | Select -ExpandProperty browser_download_url -First 1
-          .\wingetcreate.exe update Cryptomator.Cryptomator -s -v $github.release.tag_name -u $installerUrl -t ${{ secrets.CRYPTOBOT_WINGET_TOKEN }}
-        shell: pwsh
+    if: github.event.action == 'published'
+    uses: ./.github/workflows/winget.yml
+    with:
+      releaseTag: ${{ github.event.release.tag_name }}
+    secrets: inherit
```

Comment:
what does `secrets: inherit` do?

Reply:
make all secrets from caller-workflow accessible.

Reply:
since the target workflow is in the same repo, doesn't it have access to `${{ secrets.CRYPTOBOT_WINGET_TOKEN }}` anyway?

in other words: is this required? have you tested without?

Reply:
> doesn't it have access to ${{ secrets.CRYPTOBOT_WINGET_TOKEN }} 

Exactly my thinking. I wanted to ensure that directly at the first real run it is successful, hence i decided for the inherit flag.

Reply:
I'll remove it.

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/2471/commits/69747a7a77ffb805485865dbf772e6eed4a7655f